<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="inputString">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputString">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="InputModul" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>371</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="inputBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HEX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEZ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TERN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OCT</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="outputBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HEX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEZ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TERN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OCT</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>getInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
